inspire/config.py

```diff
@@ -243,6 +244,39 @@
 
 REMEMBER_COOKIE_SECURE = True
 
+# Inspire specific config
+
+INSPIRE_EMAIL_REGEX = "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
+INSPIRE_URL_REGEX = "^(?:http(?:s)?:\/\/)?(?:www\.)?(?:[\w-]*)\.\w{2,}$"
+
+INSPIRE_AFFILIATION_RANKS = ['senior', 'junior', 'staff', 'visitor', 'postdoc',
+                             'phd', 'masters', 'undergrad']
+
+# year - month (2 digits) -day (2 digits)
+INSPIRE_DATE_REGEX = "^[1|2][0-9]{3}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0|1])$"
+
+INSPIRE_AFFILIATION_STATUS = ['current']
+INSPIRE_EXPERIMENT_STATUS = ['current']
+INSPIRE_EMAIL_STATUS = ['current']
+
+INSPIRE_NAME_NUMERATIONS = "^Jr\.|Sr\.|I{1,3}|IV|VI{0,3}$"
+INSPIRE_NAME_STATUSES = ['active', 'deceased', 'departed', 'retired']
+INSPIRE_NAME_TITLES = ['Sir']
```

Comment:
Which should stay here in your opinion? That's true that I should not move all of the variables here.